homeassistant/components/ista_ecotrend/strings.json

```diff
@@ -38,10 +38,12 @@
   "entity": {
     "sensor": {
       "heating": {
-        "name": "Heating"
+        "name": "Heating",
+        "unit_of_measurement": "units"
       },
       "heating_cost": {
-        "name": "Heating cost"
+        "name": "Heating cost",
+        "unit_of_measurement": "EUR"
```

Comment:
This is a currency symbol, why does it need translating?

../Frenck

Reply:
Ah, ok, wasn't sure if that goes now into translations or not, I'll revert it